### DISCUSSION TITLE
Mounten eines Fritz!Nas Volumes nicht möglich

### DISCUSSION THREAD

**Discussion by MaHe801310** [OpenCCU/OpenCCU#2372] [🇩🇪 Fragen&Antworten] [answered]
### Describe the issue you are experiencing

Der Testversuch des Einhängens eines Fritz!Nas-Ordners mittels der bekannten, dokumentierten & angepassten Methode

mount -t cifs -o noserverino,vers=3.11,username=USER,password=PASS //192.168.1.1/fritz.nas/test/RM_B /mnt

schlägt fehl.

Auch mittels angepasstem Pfad unter aktuellem Fritz!OS 7.52

mount -t cifs -o noserverino,vers=3.11,username=USER,password=PASS //192.168.1.1/nas#/files//RM_B /mnt

schlägt der Versuch mittels Fehlermeldung "No such file or directory" fehl.

Scheinbar wird der Pfad bei der Kommandoausführung nicht komplett übernommen bzw. ausgewertet.
Als Indiz hierfür werte ich den wiedergegebenen, unvollständigen Pfad im Bezug der genannten Fehlermeldung.

Offensichtlich wird das Rautezeichen im übergebenen Pfad nicht erkannt. Auch ein setzen des Zeichens, oder auch
des ganzen Pfades in Hochkomma (') trägt nicht zur Lösung bei.

Auf Sonderzeichen  in Benutzernamen und Passwort wurde bewusst verzichtet.

### Describe the behavior you expected

Das Mounten von Fritz!Nas-Verzeichnissen sollte funktionieren.

### Steps to reproduce the issue

1. SSH unter RM aktivieren
2. Login via SSH-Client auf RM (Windows 10 SSH-Befehl)
3.  Angepasste Kommandoeingabe nach HowTo-Syntax
4. Fehlermeldung
...

### What is the version this bug report is based on?

3.69.7-20230506

### Which base platform are you running?

rpi3 (RaspberryPi3)

### Which HomeMatic/homematicIP radio module are you using?

RPI-RF-MOD

### Anything in the logs that might be useful for us?

```txt
Da Kommandozeileneingabe, kein Log.
```

**Comment by Baxxy13** [comment, marked as answer]
> Das Mounten von Fritz!Nas-Verzeichnissen sollte funktionieren.

`/bin/mount -t cifs -o noserverino,vers=3.11,username=USER,password=PASS //192.168.1.1/FRITZ.NAS/test/RM_B /mnt`

Funktioniert bei meiner 6490 mit FW: 7.51 

Letztlich musste `fritz.nas` nur in `FRITZ.NAS` geändert werden.

Ist aber kein RaspberryMatic-Problem.

**Reply by MaHe801310** [reply]
Wow, was ist ein Mensch bei gewissen Dingen mit der Zeit aber auch vergesslich ;-)

Ich hab' mich bei der Syntax zu arg von der FRITZ!Nas-Verzeichnisstruktur (Oberfläche Box => Fritz!Nas =>.........)
verleiten  lassen.

Darüber entfiel mir auch, dass ich den Namen des SMB-Shares von Standard (FRITZ.NAS) auf eine Alternativbezeichnung geändert hatte.

JaJa....., zuviel Arbeit und Bilder machen blind.

Vielen Dank niochmals für eure Bemühungen; ich konnte den Fehler aufdecken und mittels der hier veröffentlichten Syntax das Fritz!Nas-Share erfolgreich in /mnt einbinden.

Vielen Dank nochmals den Beteiligten.

**Reply by jens-maus** [reply]
Wunderbar das du es lösen konntest. Für die Zukunft merken: Bevor du ein Issue Ticket hier aufmachst erst einmal deine Problemchen hier oder im homematic forum zur diskussion stellen und erst WENN sich darauf ergibt das das ein allgemeingültiges Problem wohl ist dann ein entsprechende Issue Ticket aufmachen.

**Reply by Baxxy13** [reply]
Bitte nochmal ein konkretes Beispiel zeigen das wir dann mit ins Wiki aufnehmen können.
Das Klein- vs. Großschreibung (beim Fritz-Nas) hat mich tatsächlich selbst überrascht. 
Ich mounte inzwischen auf einen Openmediavault-Share.

**Comment by jp112sdl** [comment]
Bei mir ebenfalls fehlerfrei an der 6660 mit FW 7.56

> Ist aber kein RaspberryMatic-Problem.

+1

**Reply by jens-maus** [reply]
Ich schätze das gleich ein und habe das ganze hier deshalb man in den Discussions Bereich verschoben.
